test/test_subdir_package.py

```diff
@@ -35,7 +35,7 @@ def test(capfd, tmp_path):
     project_dir = tmp_path / 'project'
     subdir_package_project.generate(project_dir)
 
-    package_dir = os.path.join(project_dir, 'src', 'spam')
+    package_dir = os.path.join('src', 'spam')
```

Comment:
Not sure I agree with the spirit of this change - package_dir is relative to cwd, but cwd is changed within cibuildwheel_run. It should work either way, right? If so, the absolute path is more correct in my eyes

Reply:
`cwd` is changed in the subprocess before the cibuildwheel is called. I hadn't realised that the default is relative, but the full path just makes more sense to me... not a big deal though if this change is important for some other reason.